### ISSUE TITLE
widget name field paths [I18N]

### ISSUE THREAD

**Issue by aphillips** [w3c/miniapp#105]
https://w3c.github.io/miniapp/specs/manifest/#widgets

> Widgets are a part of a MiniApp. Specifically, a MiniApp package can include MiniApp Pages and Widgets concurrently. As a part of the MiniApp application, widgets share some of the Manifest fields with the MiniApp main program, e.g. dir, lang, appID, appName, shortName, icons, versionName and versionCode. However, a widget also has its private fields, as shown in the following table: 

Widget has a `name ` field in the table that follows. What is the character set, length limit/content limits (if any) for this field? Can base direction be set for the widget?

Similarly there is a `path` field. Should this be understood to be a URL path?

**Comment by zhangyongjing** [comment]
> https://w3c.github.io/miniapp/specs/manifest/#widgets
> 
> > Widgets are a part of a MiniApp. Specifically, a MiniApp package can include MiniApp Pages and Widgets concurrently. As a part of the MiniApp application, widgets share some of the Manifest fields with the MiniApp main program, e.g. dir, lang, appID, appName, shortName, icons, versionName and versionCode. However, a widget also has its private fields, as shown in the following table:
> 
> Widget has a `name ` field in the table that follows. What is the character set, length limit/content limits (if any) for this field? Can base direction be set for the widget?
> 
> Similarly there is a `path` field. Should this be understood to be a URL path?

As already mentioned, Widget shares the `dir`, `lang` configuration of the whole MiniApp. 

And yes, `path` is a URL-like string. We can clarify it in the next update.

**Comment by xfq** [comment]
> As already mentioned, Widget shares the dir, lang configuration of the whole MiniApp.

I don't think we made this clear in the [description](https://w3c.github.io/miniapp/specs/manifest/#dir) of `dir`:

> `dir` specifies the base direction of the text value of the app name, short name and description members.

So we should clarify that.

**Comment by zhangyongjing** [comment]
will do soon.

**Comment by zhangyongjing** [comment]
See update in #107

**Comment by zhangyongjing** [comment]
Can we close this now? @aphillips @xfq
